.buildkite/scripts/run_deploy_release.sh

```diff
@@ -38,6 +38,7 @@ elif [[ "${RUN_TYPE}" == "release" ]]; then
     $mvnw_command release:prepare release:perform \
       --settings .buildkite/mvn-settings.xml \
       -Darguments="-DskipTests --settings .ci/settings.xml" \
```

Comment:
Should we get rid of the `--settings .ci/settings.xml` here as well? I don't know / don't think it's needed...